src/Controllers/ExampleController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace App\Controllers;
+
+
+use App\Models\ExampleModel;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Slim\Http\Response;
+
+class ExampleController
+{
+    protected ExampleModel $model;
+
+    public function __construct(ExampleModel $model)
+    {
+        $this->model = $model;
+    }
+
+    public function __invoke(RequestInterface $request, Response $response): ResponseInterface
```

Comment:
Why is the response param type hinted to `Slim\Http\Response` instead of using the interface: `Psr\Http\Message\ResponseInterface`?